src/StatementProvider.js

```diff
@@ -0,0 +1,69 @@
+( function ( wb, $ ) {
+	'use strict';
+
+	/**
+	 * Combination of a claim, a rank and references.
```

Comment:
This describes a single statement, but this provider is for something that contains many statements, right?